biorbd_optim/dynamics.py

```diff
@@ -63,6 +63,32 @@ def forward_dynamics_torque_muscle_driven(states, controls, nlp):
         qddot_reduced = nlp["q_dot_mapping"].reduce.map(qddot)
         return vertcat(qdot_reduced, qddot_reduced)
 
+    @staticmethod
+    def forward_dynamics_excitation_driven(states, controls, nlp):
+        q, qdot, qdot_reduced, residual_tau = Dynamics.__dispatch_data(states, controls, nlp)
+
+        muscles_states = biorbd.VecBiorbdMuscleStateDynamics(nlp["nbMuscle"])
+        muscles_excitation = controls[nlp["nbTau"]:]
+        muscles_activations = states[nlp["nbQ"] + nlp["nbQdot"]:]
+
```

Comment:
I am not sure a space is required here, it feels that the for section is closely related to the initiation of these variables